star-ui/src/components/Nav2GoalPanel.tsx

```
@@ -0,0 +1,124 @@
+
+                {/* Goal inputs */}
+                <div style={{ display: 'grid', gridTemplateColumns: '1fr 1fr 1fr', gap: '8px' }}>
+                    {[
+                        { label: 'X (m)', value: goalX, set: setGoalX },
+                        { label: 'Y (m)', value: goalY, set: setGoalY },
+                        { label: 'θ (rad)', value: goalTheta, set: setGoalTheta },
+                    ].map(f => (
+                        <div key={f.label}>
+                            <label style={{ fontSize: '9px', color: 'rgba(255,255,255,0.3)', textTransform: 'uppercase', display: 'block', marginBottom: '3px' }}>
+                                {f.label}
+                            </label>
```

Biome (2.4.4)

[error] A form label must be associated with an input.

(lint/a11y/noLabelWithoutControl)